Jint/Native/Number/NumberPrototype.cs

```diff
@@ -234,15 +235,15 @@ public static string ToBase(long n, int radix)
                 return "0";
             }
 
-            var result = new StringBuilder();
+            var result = StringBuilderPool.GetInstance();
```

Comment:
Would it make sense to have the pool return a disposable instance and release it on disposal?